internal/traceprof/traceproftest/go.mod

```diff
@@ -58,3 +58,5 @@ replace github.com/DataDog/dd-trace-go/v2 => ../../..
 replace github.com/DataDog/dd-trace-go/v2/contrib/google.golang.org/grpc => ../../../v2/contrib/google.golang.org/grpc
 
 replace github.com/DataDog/dd-trace-go/v2/contrib/julienschmidt/httprouter => ../../../v2/contrib/julienschmidt/httprouter
+
+replace github.com/DataDog/dd-trace-go/v2/contrib/net/http => ../../../v2/contrib/net/http
```

Comment:
Wouldn't a `go.work` file remove the need for this and all members of its species?

Reply:
Possibly, yeah. I'll ping @darccio about that.

Reply:
@RomainMuller @knusbaum I did a quick test. A `go.work` file does help to remove the need for `replace` directives.

Let's implement it in a separate PR.